frontend/packages/helm-plugin/src/components/list-page/HelmChartRepositoryRow.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { LazyActionMenu } from '@console/dynamic-plugin-sdk/src/lib-internal';
+import { RowFunctionArgs, TableData } from '@console/internal/components/factory';
+import { ResourceLink, Timestamp, Kebab } from '@console/internal/components/utils';
+import { referenceFor, referenceForModel } from '@console/internal/module/k8s';
+import { HelmChartRepositoryModel } from '../../models';
+
+const revisionReference = referenceForModel(HelmChartRepositoryModel);
+
+const HelmChartRepositoryRow: React.FC<RowFunctionArgs> = ({ obj }) => {
+  const { t } = useTranslation();
+  const objReference = referenceFor(obj);
+  const context = { [objReference]: obj };
+  return (
+    <>
+      <TableData>
+        <ResourceLink
+          kind={revisionReference}
+          name={obj.metadata.name}
+          namespace={obj.metadata?.namespace}
+        />
+      </TableData>
+      <TableData>{obj.spec?.name ?? '-'}</TableData>
+      <TableData>{t('helm-plugin~All namespaces')}</TableData>
+      <TableData>{obj.spec?.disabled ?? '-'}</TableData>
```

Comment:
The other places uses a translation here:

```suggestion
      <TableData>{obj.spec?.disabled ? t('helm-plugin~True') : t('helm-plugin~False')}</TableData>
```